src/components/page-configuration/overview.tsx

```diff
@@ -0,0 +1,112 @@
+import { List, Tooltip, Typography } from '@equinor/eds-core-react';
+import { AuthCodeMSALBrowserAuthenticationProvider } from '@microsoft/microsoft-graph-client/authProviders/authCodeMsalBrowser';
+import * as PropTypes from 'prop-types';
+import { useEffect, useRef, useState } from 'react';
+import { AsyncState } from '../../effects/effect-types';
+import { RequestState } from '../../state/state-utils/request-states';
+
+import Alert from '../alert';
+import SimpleAsyncResource from '../async-resource/simple-async-resource';
+import { useAuthentication } from '../graph/authentication';
+import { getGroup } from '../graph/graphService';
+
+interface OverviewProps {
+  adGroups?: string[];
+  appName: string;
+}
+
+export const Overview = ({ adGroups, appName }: OverviewProps): JSX.Element => {
+  const auth = useAuthentication();
+  const mountedRef = useRef(true);
+
+  const [result, setResult] = useState<AsyncState<Array<React.ReactElement>>>({
+    data: undefined,
+    status: RequestState.IN_PROGRESS,
+  });
+
+  const getGroupInfo = (
+    authProvider: AuthCodeMSALBrowserAuthenticationProvider,
+    groups: Array<string>
+  ) => {
+    try {
+      const data: React.ReactElement[] = [];
+      const groupResult = groups.map(async (id) => {
+        await getGroup(authProvider, id).then((group) =>
+          data.push(
+            <List.Item key={group.id}>
+              <Typography
+                link
+                href={`https://portal.azure.com/#blade/Microsoft_AAD_IAM/GroupDetailsMenuBlade/Overview/groupId/${group.id}`}
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                {group.displayName}
+              </Typography>
+            </List.Item>
+          )
+        );
+      });
+      Promise.all(groupResult).then(() => {
+        if (!mountedRef.current) return null;
+        setResult({ data: data, status: RequestState.SUCCESS });
+      });
+    } catch (err) {
+      setResult({ data: undefined, status: RequestState.FAILURE });
+    }
+  };
+
+  useEffect(() => {
+    mountedRef.current = true;
+    if (adGroups?.length > 0) {
+      if (auth.authProvider) {
+        getGroupInfo(auth.authProvider, adGroups);
+      }
+    } else {
+      setResult({
+        data: undefined,
+        status: RequestState.SUCCESS,
+      });
+    }
+    return () => {
+      mountedRef.current = false;
+    };
+  }, [adGroups, auth?.authProvider]);
```

Comment:
missing effect dependency
```suggestion
  }, [adGroups, auth?.authProvider, getGroupInfo]);
```